src/components/main-layout/RightPanel.tsx

```diff
@@ -63,7 +63,8 @@ const RightPanel: React.FC<RightPanelProps> = ({
         >
             <VisibilityControl notInIntroduction>
                 <h3>
-                    Concentration over time for <AB name={productName} />{" "}
+                    Concentration over time for{" "}
+                    {productName === ProductName.AB ? <AB /> : <AC />}{" "}
```

Comment:
Are there only these two ProductNames? What if more are added in the future?

Reply:
no, the design for the app is complete, and there won't ever be other products